### ISSUE TITLE
Application Domain

### ISSUE THREAD

**Issue by ajburls** [dwyl/best-evidence#30]
As product owner
I need to have a domain where the project can be hosted following the apps development
So that the project is independent

**Comment by markwilliamfirth** [comment]
@ajburls Here are some suggested available domains:

bestevidenceapp.com
bestevidenceapp.co.uk
bestevidence.co

**Comment by ajburls** [comment]
Thanks for these suggestions. When I was working on this app before DWYL's involvement, the developers bought the domain name "BestEvidence.Info" for the app.

I have written to Shakur to establish the status of this domain. Would you think this a suitable name. If this name is lost to us then we can choose one of the ones you have suggested. I probably like last as shortest best but do not have strong feelings.

**Comment by markwilliamfirth** [comment]
@ajburls 

Your question via email:

>I seem to remember that this is initially being hosted by your company (for a year was it?)  Please confirm this is correct. Otherwise I will need to make alternative arrangements for hosting.

The proposal says:

>Hosting costs cover 6 months of heroku with 1 basic dyno ($7 USD a month), 1 hobby-basic tier for additional database backups plus $15 a month expedited SSL certificate. We expect the 6 months to begin at the end of sprint 1. [£260 is quoted cost]

Would you like us to update this to 1 year of hosting? We have yet to invoice for hosting costs so this could be done easily (although obviously the price for the year will be larger)

**Comment by ajburls** [comment]
Probably better to make your hosting shorter if anything as we need it to be hosted on City servers if we are going to do research with the app, so the sooner that happens the better.

**Comment by markwilliamfirth** [comment]
@ajburls I have to follow the original proposal so we'll have to stick with what we have in the proposal and we can move the application whenever it is necessary to do so.

Next steps:

- [ ] Mark to contact Amanda's domain contact to retrieve domain

- [ ] Transfer of domain

- [ ] Mark to link domain to production site

**Comment by markwilliamfirth** [comment]
@ajburls please let me know if you would like this completed during this sprint (Sprint 2) or Sprint 3

**Comment by ajburls** [comment]
@markwilliamfirth This can be Sprint 3 as it is behind the scenes and will not be apparent to user and priority should be to get something good we can show to people about what the app will be like in use.

**Comment by markwilliamfirth** [comment]
This has almost been completed, waiting on approval of final confirmation email by Shakur

Once this has been approved http://bestevidence.info/ will link to best-evidence-app.herokuapp.com

**Comment by markwilliamfirth** [comment]
@ajburls this has now been completed: http://www.bestevidence.info/

**Comment by markwilliamfirth** [comment]
@ajburls please close this if you are happy that the domain has been connected

**Comment by ajburls** [comment]
@markwilliamfirth Mark, I didn't close this because I was unsure about the issue of security. Will this be as secure as an Https domain as it links to the https domain. It needs to be. I know we have another issue about what final standards we need and I am awaiting advice from within City but it would be good to be reassured that it has this minimum level

**Comment by markwilliamfirth** [comment]
@ajburls this issue is just for connecting the domain and application. The security issue will be dealt with here: #193

**Comment by ajburls** [comment]
@markwilliamfirth Thanks
